api/src/repositories/survey-critter-repository.ts

```diff
@@ -51,27 +51,41 @@ export class SurveyCritterRepository extends BaseRepository {
    * @returns {*}
    * @member SurveyRepository
    */
-  async removeCritterFromSurvey(surveyId: number, critterId: number): Promise<number> {
-    defaultLog.debug({ label: 'removeCritterFromSurvey', surveyId });
-    const queryBuilder = getKnex().table('critter').delete().where({ survey_id: surveyId, critter_id: critterId });
+  async removeCritterFromSurvey(critterId: number): Promise<number> {
+    defaultLog.debug({ label: 'removeCritterFromSurvey', critterId });
+    const queryBuilder = getKnex().table('critter').delete().where({ critter_id: critterId });
     const response = await this.connection.knex(queryBuilder);
     return response.rowCount;
   }
 
   /**
-   * Add a deployment to the critter.
+   * Will insert a new critter - deployment uuid association, or update if it already exists.
    *
    * @param {number} critterId
    * @param {string} deplyomentId
-   * @return {*}  {Promise<number>}
+   * @returns {*}
    * @memberof SurveyCritterRepository
    */
-  async addDeployment(critterId: number, deplyomentId: string): Promise<number> {
+  async upsertDeployment(critterId: number, deplyomentId: string): Promise<number> {
```

Comment:
I think Postgres has syntax for perfomring an insert or update automoatically. I think the it's something like `INSERT INTO [...] ON CONFLICT DO NOTHING`... But there are other spots in the codebase you could borrow from, I think